src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -26,7 +26,7 @@ const Body = BaseForm.extend({
         View.extend({
           tagName: 'section',
           className: 'password-authenticator--rules',
```

Comment:
Btw, I'm not sure if there's a way to use `data-se` here - https://github.com/okta/okta-signin-widget/pull/1273#discussion_r449218779 since it's a `View`..

Reply:
@santhoshbalakrishnan

Reply:
iirc, 

```
attributes: {
  'data-se': 'xxx'
}
```

look for `framework/View.js` in okta-ui